scripts/buildtools/commands/verify-package.js

```diff
@@ -33,7 +33,10 @@ exports.handler = function() {
     'dist/labels/json/login_ru.json',
     'dist/sass/_fonts.scss',
     'dist/font/okticon.ttf',
-    'dist/font/okticon.woff'
+    'dist/font/okticon.woff',
+    'types/index.d.ts',
+    'types/generated/src/index.d.ts',
+    'types/generated/packages/@okta/courage-dist/types/courage/framework/Model.d.ts'
```

Comment:
is this just a sanity check that `courage for widget` types has been generated?

Reply:
yes, verify that:
- types were copied to dist
- widget generated types exist
- courage generated types exist